lib-multisrc/mayotune/src/eu/kanade/tachiyomi/multisrc/mayotune/MayoTune.kt

```diff
@@ -0,0 +1,139 @@
+    override fun latestUpdatesNextPageSelector(): String? = throw UnsupportedOperationException()
+    override fun latestUpdatesSelector(): String = throw UnsupportedOperationException()
+    override fun latestUpdatesFromElement(element: Element): SManga =
+        throw UnsupportedOperationException()
+
+    // Search
+    override fun fetchSearchManga(
+        page: Int,
+        query: String,
+        filters: FilterList,
+    ): Observable<MangasPage> {
+        val mangas = mutableListOf<SManga>()
+        sourceList.map {
+            if (it.title.lowercase().contains(query.lowercase())) {
+                mangas.add(it)
+            }
+        }
+        return Observable.just(MangasPage(mangas, false))
+    }
+
+    override fun searchMangaRequest(page: Int, query: String, filters: FilterList) =
+        throw UnsupportedOperationException()
+
+    override fun searchMangaNextPageSelector() = throw UnsupportedOperationException()
+    override fun searchMangaSelector() = throw UnsupportedOperationException()
+    override fun searchMangaFromElement(element: Element) = throw UnsupportedOperationException()
+
+    // Get Override
+    override fun mangaDetailsRequest(manga: SManga): Request {
+        return GET(manga.url, headers)
+    }
+
+    override fun chapterListRequest(manga: SManga): Request {
+        return GET(manga.url + "/api/chapters", headers)
+    }
+
+    override fun pageListRequest(chapter: SChapter): Request {
+        return GET(chapter.url, headers)
+    }
+
+    // Details
+
+    override fun mangaDetailsParse(document: Document): SManga = SManga.create().apply {
+        val statusText =
+            document.selectFirst("div.text-center:contains(Status)")?.text()
+                ?.substringBefore("Status")
+                ?.trim()
+
+        url = sourceList.first().url
+        title = sourceList.first().title
+        artist = sourceList.first().artist
+        author = sourceList.first().author
+        description = document.selectFirst(".text-lg")?.text()
+        genre = document.selectFirst("span.text-sm:nth-child(2)")?.text()?.replace("•", ",")
+        status = when (statusText) {
+            "Ongoing" -> SManga.ONGOING
+            "Completed" -> SManga.COMPLETED
+            "Cancelled" -> SManga.CANCELLED
+            "Hiatus" -> SManga.ON_HIATUS
+            "Finished" -> SManga.PUBLISHING_FINISHED
+            else -> SManga.UNKNOWN
+        }
+        thumbnail_url = document.selectFirst("img.object-contain")?.absUrl("src")
+            ?.ifEmpty { sourceList.first().thumbnail_url }
+    }
+    // Chapters
+
+    override fun chapterListSelector() = throw UnsupportedOperationException()
+    override fun chapterFromElement(element: Element): SChapter =
+        throw UnsupportedOperationException()
+
+    override fun chapterListParse(response: Response): List<SChapter> {
+        val chapters = json.decodeFromString<List<ChapterDto>>(response.body.string())
```

Comment:
```suggestion
        val chapters = response.parseAs<List<ChapterDto>>()
```
need to import, also remove the json instance declared near the top